server/cache.go

```diff
@@ -36,25 +36,19 @@ var (
 )
 
 type clusterInfo struct {
+	*schedule.BasicCluster
 	sync.RWMutex
```

Comment:
The RWMutex should also wrap the basic cluster, if yes, move the RWMutex to the first line of the clusterInfo.